src/TagLib/Ogg/XiphComment.cs

```diff
@@ -1123,23 +1277,23 @@ IEnumerator IEnumerable.GetEnumerator()
 		///    <para>This property is implemented using the COVERART
 		///    field.</para>
 		/// </remarks>
-		public override IPicture [] Pictures {
-			get {
-				string[] covers = GetField ("COVERART");
-				IPicture[] pictures = new Picture[covers.Length];
-				for (int ii = 0; ii < covers.Length; ii++) {
-					ByteVector data = new ByteVector (Convert.FromBase64String (covers[ii]));
-					pictures[ii] = new Picture (data);
+		public override IPicture[] Pictures
+		{
```

Comment:
The braces here should be on the same line. Similar to how the replaced code looks.

Reply:
Formatting fixed in edb2d97093ebabd8f21ceb1d9e21406a295ca688